packages/compiler/lib/lib.cadl

```diff
@@ -1,8 +1,31 @@
 namespace Cadl;
 
+@intrinsic("any")
```

Comment:
I think these should be compiler intrinsics like `never` rather than models. This works-ish? But runs the risk that e.g. emitters will treat them like regular models which will probably have very bad effects.

Reply:
I also would like to take this opportunity to delete Map from intrinsics, do we still need that?

Reply:
do we not want to make the distincion between a `Record<T>` and a `Map<K, V>`